.env

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-export ST2_VERSION=3.2.0
+export ST2_VERSION=3.7.0
```

Comment:
Is this the most recent version available? It would be ideal to use 3.8.

Reply:
haha good point, it was actually the latest version when I made these changes, I'll update this